internal/cli/cli.go

```diff
@@ -73,14 +72,18 @@ func (c *Cli) Execute(controller controller.ControllerInterface) {
 	}
 }
 func createInitCommand(controller controller.ControllerInterface) *cobra.Command {
-	return &cobra.Command{
+	initCmd := &cobra.Command{
 		Use:   "init",
 		Short: "Login to an existing KintoHub account",
 		Long:  `Helps you to log in an already existing KintoHub account`,
+		Args:  cobra.ExactArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
 			controller.Init(args[0])
 		},
 	}
+	initCmd.SetUsageTemplate("\nUsage:\nSet new master host:\n\t" +
```

Comment:
comment - can (`
  Usage:....
  kinto init [....
`)

Work? instead of doing \n\t?

Reply:
Technically it will work but won't look that nice